backend/src/main/java/com/benine/backend/http/RequestHandler.java

```diff
@@ -45,6 +45,22 @@ public Attributes parseURI(String uri) throws MalformedURIException {
   }
 
   /**
+   * Formats the response message for a success or failure.
+   * @param exchange the HttpExchange.
+   * @param correct boolean that is true if the exchange is successful. False otherwise.
+   */
+  public void responseMessage(HttpExchange exchange, boolean correct) {
```

Comment:
Now you essentially have one method with two different types of behaviour specifying which behaviour you want by the boolean as argument. I think its cleaner to devide this in two methods. However if you and the others don't agree with this. You can atleast just make it:
 ...{
    respond(exchange, response = "{\"succes\":\""+correct+"\"}";  
   }